### ISSUE TITLE
Release Pillow 6.2.2 on January 2, 2020

### ISSUE THREAD

**Issue by radarhere** [python-pillow/Pillow#4328]
## Purpose

This is a security release. This is the last release to support Python 2.7.

## Point Release

Released as needed for security, installation or critical bug fixes.

* [x] Make necessary changes in `master` branch.
* [x] Update `CHANGES.rst`.
* [x] Check out release branch e.g.:
  ```bash
  git checkout -t remotes/origin/5.2.x
  ```
* [x] Cherry pick individual commits from `master` branch to release branch e.g. `5.2.x`.
* [x] Check [Travis CI](https://travis-ci.org/python-pillow/Pillow) to confirm passing tests in release branch e.g. `5.2.x`.
* [x] In compliance with [PEP 440](https://www.python.org/dev/peps/pep-0440/), update version identifier in `src/PIL/_version.py`
* [x] Run pre-release check via `make release-test`.
* [x] Create tag for release e.g.:
  ```bash
  git tag 5.2.1
  git push --tags
  ```
* [x] Create source distributions e.g.:
  ```bash
  make sdist
  ```

* [x] Create [binary distributions](https://github.com/python-pillow/Pillow/blob/master/RELEASING.md#binary-distributions)

### Windows
* [x] Contact `@cgohlke` for Windows binaries via release ticket e.g. https://github.com/python-pillow/Pillow/issues/1174.
* [x] Download and extract tarball from `@cgohlke` and `twine upload *`.

### Mac and Linux
* [x] Use the [Pillow Wheel Builder](https://github.com/python-pillow/pillow-wheels):
  ```bash
  git clone https://github.com/python-pillow/pillow-wheels
  cd pillow-wheels
  ./update-pillow-tag.sh [[release tag]]
  ```
* [x] Download distributions from the [Pillow Wheel Builder container](http://a365fff413fe338398b6-1c8a9b3114517dc5fe17b7c3f8c63a43.r19.cf2.rackcdn.com/).
  ```bash
  wget -m -A 'Pillow-<VERSION>-*' \
  http://a365fff413fe338398b6-1c8a9b3114517dc5fe17b7c3f8c63a43.r19.cf2.rackcdn.com
  ```

---

* [x] Upload all binaries and source distributions e.g. `twine upload dist/Pillow-5.2.0*`

* [x] Create a [new release on GitHub](https://github.com/python-pillow/Pillow/releases/new)

## Publicize Release

* [x] Announce release availability via [Twitter](https://twitter.com/pythonpillow) e.g. https://twitter.com/PythonPillow/status/1013789184354603010

## Documentation

* [x] Make sure the [default version for Read the Docs](https://pillow.readthedocs.io/en/stable/) is up-to-date with the release changes

**Comment by cgohlke** [comment]
I built Windows binaries for [Pillow-6.2.2](https://www.lfd.uci.edu/~gohlke/download/Pillow-6.2.2-win-dist.zip) and [Pillow-7.0.0](https://www.lfd.uci.edu/~gohlke/download/Pillow-7.0.0-win-dist.zip) in case you still need them.

**Comment by radarhere** [comment]
Yes @cgohlke, thanks for being proactive! If you could just assure me that you've included the 'Release notes for 6.2.2' commits that I added after the version bump, then yes, please upload the files.

**Comment by hugovk** [comment]
Thank you very much @radarhere for making this double release, and @cgohlke for the binaries!
